main.py

```diff
@@ -349,14 +349,16 @@ def journal_results(activities, events):
 
     # Grab any dynamic visualizations that were uploaded.
     # TODO: This could be dynamically looped by listing attachments instead of hardcoding.
-    try:
-        spec2 = LAMP.Type.get_attachment(participant, "lamp.dashboard.experimental.activity_segmentation")["data"]
-    except LAMP.ApiException:
-        spec2 = []
-    try:
-        spec3 = LAMP.Type.get_attachment(participant, "lamp.dashboard.experimental.sensor_data_quality.3hr")["data"]
-    except LAMP.ApiException:
-        spec3 = []
+    # 5/3/21 Temporarly removed due to formating issues and lack of updates
```

Comment:
No need to leave a date-stamp here as Git already shows a blame of when changes were made to what lines!

Reply:
👍